src/R/Core/Impl/Microsoft.R.Core.csproj

```diff
@@ -79,6 +80,12 @@
     <Compile Include="AST\Utility\AstWriter.cs" />
     <Compile Include="AST\Values\MissingValue.cs" />
     <Compile Include="AST\Values\NullValue.cs" />
+    <Compile Include="Install\IMicrosoftRClientInstaller.cs" />
+    <Compile Include="Install\ISupportedRVersionRange.cs" />
```

Comment:
`R.Support` depends on `R.Host.Client`. I need it to be in some place that doesn't, so that broker can reference that, and `Host.Client` can reference broker.